spec/lib/tasks/bulk_import_documents_from_csv_spec.rb

```diff
@@ -39,6 +39,10 @@
   let(:task) { Rake::Task["bulk_import_documents_from_csv"] }
   let(:csv_path) { Rails.root.join("publications_export.csv") }
 
+  before(:each) do
+    allow($stdout).to receive(:write)
```

Comment:
Worth adding an inline comment saying why you're adding this (presumably to reduce noise in the output)